### PR TITLE
update hardware requirement

### PR DESCRIPTION
**Pull request by sherry-x**
### Description

Update hardware requirement for AIT3

### Test Plan

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/2662)

### CODE REVIEW

**Review by perryjrandall — APPROVED** [review]

**Review by zekun000 — APPROVED** [review]

File: developer-docs-site/docs/nodes/ait/node-requirements.md [hunk 1]
```diff
@@ -23,18 +23,49 @@ In test mode, you will be running a local network with one single node, and it s
...
+  - Azure
+      - Standard_B8ms
+
+For running an Aptos **Fullnode** we recommend the following hardware resources:
+
+  - **CPU**:
+      - 4 cores, 8 threads 
+      - 2.8GHz, or faster
+      - Intel Xeon Skylake or newer
   - **Memory**: 8GiB RAM.
```
Comment (msmouse) [inline developer-docs-site/docs/nodes/ait/node-requirements.md:49]: fullnodes won't keep up with validators with so little memory I assume.. thoughts @grao1991 ?

Comment (grao1991) [reply developer-docs-site/docs/nodes/ait/node-requirements.md:49]: I feel we should ask 16GiB.

File: developer-docs-site/docs/nodes/ait/node-requirements.md [hunk 2]
```diff
@@ -23,18 +23,49 @@ In test mode, you will be running a local network with one single node, and it s
...
+  - GCP
+      - c2-standard-8
+  - Azure
+      - Standard_B4ms
+
 ## Storage requirements
 
 The amount of data stored by the Aptos Blockhain depends on the ledger history (length) of the blockchain and the number of on-chain states (e.g., accounts). These values depend on several factors, including: the age of the blockchain, the average transaction rate and the configuration of the ledger pruner.
 
 We recommend nodes have at least 300GB of disk space to ensure adequate storage space for load testing. You have the option to start with a smaller size and adjust based upon demands. You will be responsible for monitoring your node's disk usage and adjusting appropriately to ensure node uptime.
```
Comment (grao1991) [inline developer-docs-site/docs/nodes/ait/node-requirements.md:64]: We should also ask IOPS for disk.

Comment (grao1991) [reply developer-docs-site/docs/nodes/ait/node-requirements.md:64]: In general recent years local SSD should be good enough. If people choose to use remote one on cloud, they need to:
1. Buy enough IOPS (number TBD, I feel asking for 20K-30K is not crazy, however on AWS the max of gp2/gp3 is capped at 16K, and io1/io2 is ~10x more expensive)
2. Make sure their instance type can support enough IOPS.
3. Make sure the bandwidth is enough (DB is able to consume 300M bytes per second when there is enough load)

**Pull request merged**